projects/examples/src/wells/liquidity.ts

```diff
@@ -38,12 +37,12 @@ async function main() {
   await tx.wait();
 
   // Get LP Balance
-  const lpbal = await (await well.getLPToken()).getBalance(account);
-  // const lpbal = LPTOKEN.amount(100)
-  console.log("\nLP Balance: ", lpbal.toHuman());
+  // const lpbal = await (await well.getLPToken()).getBalance(account);
+  // // const lpbal = LPTOKEN.amount(100)
+  // console.log("\nLP Balance: ", lpbal.toHuman());
```

Comment:
Is there a reason to preserve these comments? It's always going to be accessible in the git history if you delete it.

Reply:
yeah those are left commented on purpose, so you can easily do different things on your local chain by just commenting/uncommenting different parts of the files. These are just examples/local scripts used to set up tests or try things out